invenio_communities/communities/searchconfig.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is copied over from invenio-rdm-records. Should we move this logic in invenio-records-resources? @lnielsen @ntarocco